docker-compose.yml

```diff
@@ -108,6 +108,35 @@ services:
       - osm-world
       - osmboundaries
       - osmaxx-internal
+  worker-high:
+    image: geometalab/osmaxx-worker:${DEPLOY_VERSION}
+    extends:
+      file: docker-compose-common.yml
+      service: conversionbase
+    command: [honcho, -f, ./osmaxx_conversion_service/Procfile.worker, start]
+    volumes:
+      - osm_data:/var/data/osm-planet
+      - worker-data:/data/media/job_result_files
+    links:
+      - conversionserviceredis:redis
+      - world-database
+      - osmboundaries-database
+    env_file:
+      - ./compose-env/worker.env
+    environment:
+      - REDIS_HOST=redis
+      - REDIS_PORT=6379
+      - DJANGO_SETTINGS_MODULE=osmaxx_conversion_service.config.settings.worker
+      # only listens for jobs in this specific queue
+      - WORKER_QUEUES=high
+    depends_on:
+      - conversionserviceredis
+      - world-database
+      - osmboundaries-database
+    networks:
+      - osm-world
+      - osmboundaries
+      - osmaxx-internal
```

Comment:
somewhat:

```yml
  worker-base:
    image: geometalab/osmaxx-worker:${DEPLOY_VERSION}
    extends:
      file: docker-compose-common.yml
      service: conversionbase
    command: [honcho, -f, ./osmaxx_conversion_service/Procfile.worker, start]
    volumes:
      - osm_data:/var/data/osm-planet
      - worker-data:/data/media/job_result_files
    environment:
      - REDIS_HOST=redis
      - REDIS_PORT=6379
      - DJANGO_SETTINGS_MODULE=osmaxx_conversion_service.config.settings.worker
  worker:
    extends:
      file: docker-compose.yml
      service: worker-base
    volumes-from:
      - worker-base
    links:
      - conversionserviceredis:redis
      - world-database
      - osmboundaries-database
    environment:
      - REDIS_HOST=redis
      - REDIS_PORT=6379
      - DJANGO_SETTINGS_MODULE=osmaxx_conversion_service.config.settings.worker
    depends_on:
      - conversionserviceredis
      - world-database
      - osmboundaries-database
    networks:
      - osm-world
      - osmboundaries
      - osmaxx-internal
  worker-high:
    image: geometalab/osmaxx-worker:${DEPLOY_VERSION}
    extends:
      file: docker-compose.yml
      service: worker-base
    command: [honcho, -f, ./osmaxx_conversion_service/Procfile.worker, start]
    volumes:
      - osm_data:/var/data/osm-planet
      - worker-data:/data/media/job_result_files
    links:
      - conversionserviceredis:redis
      - world-database
      - osmboundaries-database
    environment:
      # only listens for jobs in this specific queue
      - WORKER_QUEUES=high
    depends_on:
      - conversionserviceredis
      - world-database
      - osmboundaries-database
    networks:
      - osm-world
      - osmboundaries
      - osmaxx-internal
```

In addition of adding more lines (longer than duplication), it also adds the issue of having a non-functioning worker-base, which always should be remembered to not be started.